__tests__/todo-config-test.ts

```diff
@@ -21,7 +21,7 @@ describe('todo-config', () => {
     it('returns default object when no package.json found', async () => {
```

Comment:
question: shouldn't it only return a default object if neither `package.json` nor `.lint-todorc.js` are found?

Reply:
If no package.json is found, I think we have real problems :)

To answer specifically, it will do both (see the next test down). It effectively falls through and returns a default config if both aren't present.

Reply:
heh agree on the first point, I guess I was just looking at the language of the test and wondering if it was too narrow. Thanks for the quick response!